#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_connection_handlers.diff by  <asterix@lagaule.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix variable name
#
# Description: fix variable name
# Author: Yann Leboulanger <asterix@lagaule.org>
# Last-Update: 2012-10-28

@DPATCH@

Index: src/common/connection_handlers.py
===================================================================
--- a/src/common/connection_handlers.py
+++ b/src/common/connection_handlers.py
@@ -1993,7 +1993,7 @@
     def _StreamCB(self, con, iq_obj):
         log.debug('StreamCB')
         gajim.nec.push_incoming_event(StreamReceivedEvent(None,
-            conn=self, stanza=obj))
+            conn=self, stanza=iq_obj))
 
     def _register_handlers(self, con, con_type):
         # try to find another way to register handlers in each class
